testProjects/networkTimeout/handler.test.js

```diff
@@ -0,0 +1,51 @@
+const iopipe = require('./iopipe.js');
+
+describe('Call to non-existant endpoint should time out after specd timeout - 100ms', () => {
```

Comment:
this reads like "specd timeout minus 100ms" if you don't mean minus, I think you can drop the specification here because it's written out in the tests.

Reply:
Nevermind, that made sense when I first scanned this, it differentiates between the two tests.